packages/pwa-kit-react-sdk/package.json

```diff
@@ -105,6 +105,7 @@
     "@loadable/component": "^5.15.0",
     "@loadable/server": "^5.15.0",
     "@loadable/webpack-plugin": "^5.15.0",
+    "@wojtekmaj/enzyme-adapter-react-17": "^0.6.6",
```

Comment:
Because this is a dev dependency and we don't require 3PP approvals for those, I think we should be ok with this. Especially with it's popularity on npm, it looks like it's commonly used. If you are feeling a little paranoid you can include this [link](https://snyk.io/advisor/npm-package/@wojtekmaj/enzyme-adapter-react-17) for the snyk report on the package. It looks like it's healthy and has no vulnerabilities.